package.json

```diff
@@ -27,11 +27,12 @@
   ],
   "scripts": {
     "lint": "jshint bin lib test",
-    "mocha": "_mocha --reporter spec test/svg-sprite.js",
+    "mocha": "_mocha --reporter spec test/**/*.spec.js test/svg-sprite.js",
```

Comment:
I'd rather be consistent here and use a common filename suffix like .test.js.

Reply:
ok, I will rename

Reply:
May I rename the `svg-sprite.js` => `svg-sprite.test.js` because of the naming convention?